starter-code/src/components/FoodBox.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+
+class FoodBox extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: 0
+    }
+  }
+  
+  render() {
+
+    const onChangeValue = (event) => {
+      if (event.target.value >= 0) {
+        this.setState({value: event.target.value})
```

Comment:
Haciendo esto, pisas el valor anterior haciendo que si quieres añadir la misma cantidad varias veces, no puedas. Para poder añadir la misma cantidad, debes reemplazar el valor actual con el resultado de la suma del actual más el nuevo valor `this.setState({ value: this.state.value + event.target.value })`